drupal/composer.json

```diff
@@ -139,9 +140,6 @@
             "drupal/graphql" : {
                 "drupal-graphql#1323: Add check for translation (from Github)": "https://patch-diff.githubusercontent.com/raw/drupal-graphql/graphql/pull/1388.patch"
             },
-            "drupal/graphql_compose" : {
-                "Add the langcode field to the menu_item entity in graphql (local patch)": "./patches/grapqhl_compose_add_langcode_to_menu_item.patch"
-            },
```

Comment:
No longer needed after this was added to graphql_compose https://git.drupalcode.org/project/graphql_compose/-/commit/12d3d19ef0c78827da673622441917a384f9ba14